docs/developer/serving/services/ingress-class.md

```diff
@@ -0,0 +1,19 @@
+# Configuring Services custom Ingress Class
+By default, when a Knative Service is created an Ingress Class (`ingress.class`) is automatically chosen for it based on the content of the `networking-config` ConfigMap located inside the `knative-serving` namespace. 
```

Comment:
```suggestion
# Configuring Services custom ingress class

By default, when a Knative Service is created, an ingress class (`ingress.class`) is automatically assigned to it based on the content of the `networking-config` ConfigMap. The `networking-config` ConfigMap is located inside the `knative-serving` namespace. 
```

slight editing suggestions for readability

Reply:
done